community/setup_in_windows.md

```diff
@@ -0,0 +1,48 @@
+# Project Set-Up In Windows Environment
+Windows users who want to run the Mathesar Docker development environment in WSL are advised to clone the repository in a Linux filesystem. 
+If you have tried running the project in windows you many ran into some issues including this 
+```
+service_1  | /usr/bin/env: ‘bash\r’: No such file or directory
+service_1  | 2022/03/30 16:20:21 Command exited with error: exit status 127
+```
+The complete guide is given below .
+## Pre-requisite
+- We assume you have Windows 10 or higher.
+- [Docker](https://docs.docker.com/get-docker/) is up and running properly in your system.
+
+# Solution 
+ - First search for "Turn Windows features on or off" , navigate to buttom and click the check-box named "Windows Subsystem For Linux".
+    
+ - Now you have to install any linux distro in your windows system .
+  Go to microsorft store and install any linux distro(Ubuntu prefered) and launch .
+ - If you do ls-la in ubuntu terminla you can see the linux files structure,
+     * so you have to access  window's files from ubuntu console, use this command to display all the storage drives avilable in windows , it will mount the storage .
+     ``
+     cd /mnt/
+     ``
+     * navigate to your desired file location e.g 
+     ```
+     cd /mnt/User/your_pc_name/Desktop/folder
+     ```
+ - Clone the repositories using command ``` git clone [repo-link](https://github.com/centerofci/mathesar)```
```

Comment:
I think this would be better as a copy-pastable separate line rather than having a web link.

Reply:
ok sir !